error.c

```diff
@@ -1066,6 +1066,7 @@ rb_bug_for_fatal_signal(ruby_sighandler_t default_sighandler, int sig, const voi
 
     if (default_sighandler) default_sighandler(sig);
 
+    ruby_default_signal(sig);
     die();
```

Comment:
`die()` may never be called?

Reply:
Yes, it's in case for some reason `sig` isn't fatal? Maybe it doesn't make sense but I wasn't 100% sure so seemed safer.